fynegen/generate/main.go

```diff
@@ -658,11 +660,11 @@ func main() {
 	ctx.UsedImports["github.com/refaktor/rye/evaldo"] = struct{}{}
 
 	var cb CodeBuilder
-	cb.Linef(`//go:build b_fyne`)
```

Comment:
You probably already plan this, but this and package would probably be nice candidate to go into confing.toml.